prep-css-syntax/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+      h1 {
+        font-family: Arial, Helvetica, sans-serif
+      }
+      p {
+        font-size:2rem
+      }
+      h3 {
+        color: firebrick
+      }
+      h2 {
+        font-style:italic
+      }
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>CSS Syntax</title>
```

Comment:
Oop! Looks like some HTML made it into your `<style></style>` element. In general we only want pure CSS code between the style tags.